resources/webroot/static/data/weathertypes_es.json

```diff
@@ -0,0 +1,35 @@
+{
+    "0": {
+        "types": [],
```

Comment:
Why is this a new file and not included in the localization files?

Reply:
We were discussing that with Fabio. I think I'll do that, there's no real need for separate files.
I just saw that there was a file for EN strings and I just "played along" 🤣

Reply:
@123FLO321 @Fabio1988 please tell me what you think of the new implementation. :)

Reply:
As discussed pls rearrange :)